scripts/prepare_sector_network.py

```diff
@@ -2426,6 +2436,9 @@ def add_rail_transport(n, costs):
     ].index  # TODO if you take nodes from the index of buses of n it's more than pop_layout
     # clustering of regions must be double checked.. refer to regions onshore
 
+    # Add location. TODO: move it into pypsa-earth
+    n.buses.location = n.buses.index

```

Comment:
When should this be moved to pypsa-earth?

Reply:
After the merge?
I believe the sector script requires some heavy changes and there is the need to update the environment too.
If you agree I open an issue on that

Reply:
I agree, please feel free to open an issue on that and then resolve this conversation

Reply:
Done